papers/Phi Fei Fo Fum (WIP)/paper.Rmd

```diff
@@ -0,0 +1,258 @@
+Thus, $\chi^2_\text{max} = N$.
+
+```{r}
+(Titanic_freq <- as.table(apply(Titanic, 2, sum)))
+p_E <- c(0.5, 0.5)
+
+cohens_w(Titanic_freq, p = p_E)
+```
+
+Unfortunately, *w* has an upper bound of 1 *only* when the variable is binomial (has two categories) and the expected distribution is uniform ($p = 1 - p = 0.5$). If the distribution is none uniform [@rosenberg2010generalized] or if there are more than 2 classes [@johnston2006measures], then $\chi^2_\text{max} > N$, and so *w* can be larger than 1.
+
+```{r}
+O <- c(90, 10)
+p_E <- c(0.35, 0.65)
+cohens_w(O, p = p_E)
+
+
+O <- c(10, 20, 80, 5)
+p_E <- c(.25, .25, .25, .25)
+cohens_w(O, p = p_E)
+```
+
+## Fei
+
+We present here a new effect size, `r "\U05E4"` (Fei), which normalizes goodness-of-fit $\chi^2$ by the proper $\chi^2_\text{max}$ for non-uniform and/or multinomial variables.
+
+The largest deviation from the expected probability distribution would occur when all observations are in the cell with the smallest expected probability. That is:
+
+$$
+p_{O} = 
+\begin{cases}
+1, & \text{if } p_i = \text{min}(p) \\
+0, & \text{Otherwise}
+\end{cases}
+$$
+
+We can find $\frac{(E_i-O_i)^2}{E_i}$ for each of these values:
+
+$$
+\frac{(p_{E}-p_{O})^2}{p_{E}} = 
+\begin{cases}
+\frac{(p_i-1)^2}{p_i} = \frac{(1-p_i)^2}{p_i}, & \text{if } p_{E} = \text{min}(p_{E}) \\
+\frac{(p_i-0)^2}{p_i} = p_i, & \text{Otherwise}
+\end{cases}
+$$
+
+Therefore, 
+
+<!-- MSB: How's this? -->
+
+$$
+\begin{split}
+\sum_{i=1}^{k}{\frac{(p_{O_i}-p_{E_i})^2}{p_{E_i}}} & = \sum_{i=1}^{k}{p_{E_i}} - \text{min}(p_{E}) + \frac{(1-\text{min}(p_{E}))^2}{\text{min}(p_{E})} \\
+& = 1 - \text{min}(p_E) + \frac{(1-\text{min}(p_E))^2}{\text{min}(p_E)} \\
+& = \frac{1-\text{min}(p_E)}{\text{min}(p_E)} \\
+& = \frac{1}{\text{min}(p_E)} - 1
+\end{split}
+$$
+
+And so, 
+
+$$
+\begin{split}
+\chi^2_\text{max} & = N \times \sum_{i=1}^{k}{\frac{(p_{O_i}-p_{E_i})^2}{p_{E_i}}} \\
+ & = N \times (\frac{1}{\text{min}(p_E)} - 1)
+\end{split}
+$$
+Finally, an effect size can be derived as: 
+
+$$
+\sqrt{\frac{\chi^2}{N \times (\frac{1}{\text{min}(p_E)} - 1)}}
+$$
+
+We call this effect size `r "\U05E4"` (Fei), which represents the voiceless bilabial fricative in the Hebrew language, keeping in line with $\phi$ (which in modern Greek marks the same sound) and $V$ (which in English marks a voiced bilabial fricative; $W$ being derived from the letter V in modern Latin alphabet). `r "\U05E4"` will be 0 when the observed distribution matches the expected one perfectly, and will be 1 when the observed values are all of the same class---the one with the smallest expected probability.
+
+```{r}
+O <- c(90, 10)
+p_E <- c(0.35, 0.65)
+fei(O, p = p_E)
+
+
+O <- c(10, 20, 80, 5)
+p_E <- c(.25, .25, .25, .25)
+fei(O, p = p_E)
+```
+
+When there are only 2 cells with uniform expected probabilities (50%), this expression reduces to $N$ and `r "\U05E4"` $= w$.
+
+```{r}
+O <- c(90, 10)
+p_E <- c(0.5, 0.5)
+
+fei(O, p = p_E)
+cohens_w(O, p = p_E)
+```
+
```

Comment:
We should add a short section to each part above demonstrating not only the tables that produce 0 and 1 but also some "small", "medium", and "large" correlations to help readers get a sense of what different values of these effect sizes look like in data.

Reply:
Not sure what you mean..?

Reply:
in addition to the real-data example, we should also demonstrate, eg, what different "benchmark" small, medium, and large correlation values (e.g., .1, .3, .5, .7, .9) look like in a contingency table

Reply:
test comment...

Reply:
test comment2